uber/templates/regform.html

```diff
@@ -92,7 +93,6 @@
     $(function () {
         staffingClicked();
         if ($.field('staffing')) {
```

Comment:
You changed the selector above to specifically find a checkbox, do we need to do the same here? I don't THINK we do but just wanna check.